netplan_cli/cli/commands/status.py

```diff
@@ -244,6 +244,28 @@ def pretty_print(self, data: JSON, total: int, _console_width=None) -> None:
                     title='Activation Mode:',
                     value=val,
                     ))
+
+            val = data.get('bridge')
+            if val:
+                pprint(('{title:>'+pad+'} {value}').format(
+                    title='Bridge:',
+                    value=val,
+                    ))
+
+            val = data.get('bond')
+            if val:
+                pprint(('{title:>'+pad+'} {value}').format(
+                    title='Bond:',
+                    value=val,
+                    ))
+
+            lst = data.get('members', [])
+            for i, val in enumerate(lst):
+                pprint(('{title:>'+pad+'} {value}').format(
+                    title='Members:' if i == 0 else '',
```

Comment:
nitpick: We should arguably call it "Interfaces", to keep it in line with the Netplan YAML nomenclature.